5.5/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM openshift/base-centos7
+
+# This image provides an Apache+PHP environment for running PHP
+# applications.
+
+MAINTAINER Ben Parees <bparees@redhat.com>
+
+# Image metadata
+ENV PHP_VERSION           5.5
+ENV IMAGE_DESCRIPTION     PHP 5.5
+ENV IMAGE_TAGS            php,php55
+ENV IMAGE_EXPOSE_SERVICES 8080:http
+#ENV PHPRC /opt/openshift/etc/php.ini
+#ENV PHP_INI_SCAN_DIR /opt/openshift/etc/php.d
+
+# Install Apache httpd and PHP
+RUN yum install -y \
+    https://www.softwarecollections.org/en/scls/rhscl/httpd24/epel-7-x86_64/download/rhscl-httpd24-epel-7-x86_64.noarch.rpm \
+    https://www.softwarecollections.org/en/scls/rhscl/php55/epel-7-x86_64/download/rhscl-php55-epel-7-x86_64.noarch.rpm && \
+    yum install -y --setopt=tsflags=nodocs httpd24 php55 php55-php php55-php-mysqlnd php55-php-pgsql php55-php-bcmath php55-php-devel \
```

Comment:
if we pull whole collection we don't need to add more modules later... I
don't see any reason to not install something here...

also it would be easier to track new packages that were added into
collection (as we install all of them)
On Apr 27, 2015 4:39 PM, "Ben Parees" <notifications@github.com> wrote:

> In 5.5/Dockerfile
> <https://github.com/openshift/sti-php/pull/1#discussion_r29151732>:
>
> > +
> > +MAINTAINER Ben Parees <bparees@redhat.com>
> > +
> > +# Image metadata
> > +ENV PHP_VERSION           5.5
> > +ENV IMAGE_DESCRIPTION     PHP 5.5
> > +ENV IMAGE_TAGS            php,php55
> > +ENV IMAGE_EXPOSE_SERVICES 8080:http
> > +#ENV PHPRC /opt/openshift/etc/php.ini
> > +#ENV PHP_INI_SCAN_DIR /opt/openshift/etc/php.d
> > +
> > +# Install Apache httpd and PHP
> > +RUN yum install -y \
> > +    https://www.softwarecollections.org/en/scls/rhscl/httpd24/epel-7-x86_64/download/rhscl-httpd24-epel-7-x86_64.noarch.rpm \
> > +    https://www.softwarecollections.org/en/scls/rhscl/php55/epel-7-x86_64/download/rhscl-php55-epel-7-x86_64.noarch.rpm && \
> > +    yum install -y --setopt=tsflags=nodocs httpd24 php55 php55-php php55-php-mysqlnd php55-php-pgsql php55-php-bcmath php55-php-devel \
>
> maybe, i was going off the v2 dep list to try to be very
> declarative/explicit. you think it matters?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/sti-php/pull/1/files#r29151732>.
>